blog/2023-06-10-conda-libmamba-solver-rollout.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Conda and the libmamba solver: Roll-out plan 2023"
+slug: "2023-06-10-conda-libmamba-solver-rollout"
+description: |
+    Learn about the upcoming rollout plan for the conda-libmamba-solver, bringing enhanced performance and user experience to conda.
+authors: [jezdez, jaimergp]
+tags: [conda-libmamba-solver, solvers, performance]
+
+---
+
+Over a year ago, the conda team at Anaconda, together with conda developers at Quansight and QuantStack, [announced the release](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) of a new solver for conda based on the libraries that the [mamba project](https://mamba.readthedocs.io/) had developed on top of the venerable libsolv project.
+
+## TLDR;
```

Comment:
```suggestion
## TL;DR
```

The official way to write it seems to be with the semicolon in the middle. https://en.wikipedia.org/wiki/TL;DR

Reply:
OK!